docs/freqai.md

```diff
@@ -534,6 +536,15 @@ for each pair, for each backtesting window within the expanded `--timerange`.
 
 ---
 
+### Hyperopt
+
+The [Hyperopt](hyperopt.md) module can be executed with some restrictions:
+
+- The `--analyze-per-epoch` hyperopt parameter is not compatible with FreqAI.
+- It's not possible to hyperopt indicators in `populate_any_indicators()` function. This means that the user cannot optimize model parameters using hyperopt. Apart from this exception, it is possible to optimize all other [spaces](hyperopt.md#running-hyperopt-with-smaller-search-space).
+- The [Backtesting](#backtesting) instructions also apply apply to Hyperopt.
```

Comment:
@robcaulk , please, could you remove the second apply word at final version? I just realized now